### ISSUE TITLE
Release Twisted 11.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5325]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#5325|
|Type|task|
|Created|2011-10-21 09:49:37Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-11.1.0-5325-2|

It's this time of the year.

<details><summary>Searchable metadata</summary>

```
trac-id__5325 5325
type__task task
reporter__therve therve
priority__normal normal
milestone__Twisted_11_1 Twisted-11.1
branch__releases_release_11_1_0_5325_2 releases/release-11.1.0-5325-2
branch_author__therve therve
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1319190577000000 1319190577000000
changetime__1321648476000000 1321648476000000
version__None None
owner__therve therve
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve removed owner|
|-|-|

Please review!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @therve|
|-|-|

  1. The news fragment for [#5062](https://github.com/twisted/twisted/issues/5062) has an extra word - _customize the their logging prefix_
  1. The fragments for [#4752](https://github.com/twisted/twisted/issues/4752) and [#4813](https://github.com/twisted/twisted/issues/4813) don't follow the conventional style

Apart from those points this looks good.  Please proceed!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [33194]) Merge release-11.1.0-5325-2

Author: therve
Reviewer: exarkun
Fixes: [#5325](https://github.com/twisted/twisted/issues/5325)

Release Twisted 11.1.0.
